### DISCUSSION TITLE
Перенос Доброграда из GM в GTA V

### DISCUSSION THREAD

**Discussion by PetrBronzo** [octothorp-team/public-dobrograd#900] [Предложения]
Я считаю что было бы неплохо создать тестовый сервер Доброграда в GTA V онлайн и посмотреть что будет, Если людям понравиться, перейти всем игрокам полностью в GTA и сделать сервер основным.

**Comment by Dawaren** [comment]
Вот только у многих людей не то что не тянет Gta V, у них ещё и может плохо тянуть Garry's mod. Так-же Gta V стоит 2.000 рублей, я не считаю что это будет оправданный перенос, считая что Chelog так и так будет скорее всего делать свою игру.

**Comment by Wani4ka** [comment]
Если перейдем на гта 5, я со своим ноутом отлетаю моментально :)

**Comment by inquizzyoff** [comment]
Таких планов у нас нет и вряд ли будут
